cirq-core/cirq/protocols/hash_from_pickle_test.py

```diff
@@ -0,0 +1,107 @@
+# Copyright 2024 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import annotations
+
+import multiprocessing
+import os
+import pathlib
+import pickle
+from collections.abc import Iterator
+from typing import Any, Hashable
+
+import pytest
+
+import cirq
+from cirq.protocols.json_serialization_test import MODULE_TEST_SPECS
+
+
+REPO_ROOT = pathlib.Path(__file__).parent.parent.parent.parent
```

Comment:
@Might be more clear if we start from `cirq` instead (also more robust against possible future refactoring)
```suggestion
REPO_ROOT = pathlib.Path(cirq.__file__).parent.parent
```

Reply:
Good point, done in 1f72d43a1ad642bc00d91bc41fc9cac5fffd9a64